src/LibraryManager/Microsoft.Web.LibraryManager.csproj

```diff
@@ -19,6 +19,9 @@
   <ItemGroup>
     <XliffResource Include="MultilingualResources\*.xlf" />
   </ItemGroup>
+  <ItemGroup>
+    <Compile Remove="Providers\Unpkg\UnpkgLibraryId.cs" />
+  </ItemGroup>
```

Comment:
Intentional?

Reply:
Yes!

---
In reply to: [195511561](https://github.com/aspnet/LibraryManager/pull/164#discussion_r195511561) [](ancestors = 195511561)

Reply:
Sorry, still not clear here. This file was deleted, right? If so, you don't need to remove it from the compilation

---
In reply to: [195566521](https://github.com/aspnet/LibraryManager/pull/164#discussion_r195566521) [](ancestors = 195566521,195511561)